lib/transport_web/controllers/backoffice_controller.ex

```diff
@@ -87,4 +88,30 @@ defmodule TransportWeb.BackofficeController do
     |> index(%{})
   end
 
+  def partners(%Plug.Conn{} = conn, params) do
+    config = make_pagination_config(params)
+    partners = Partner.list |> Scrivener.paginate(config)
+
+    conn
+    |> assign(:partners, partners)
+    |> render("partners.html")
+  end
+
+  def new_partner(%Plug.Conn{} = conn, %{"partner_url" => partner_url} = _params) do
+    with partner <- %Partner{url: partner_url},
+         true <- Partner.valid?(partner),
```

Comment:
We (I mean… *you*) should add an extra-step to verify that it is an actual url (not truncated for instance) and store the id, not the url/slug as it might change

Reply:
Done.